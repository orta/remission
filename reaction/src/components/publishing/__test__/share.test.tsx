import * as React from "react"
import * as renderer from "react-test-renderer"

import "jest-styled-components"

import Share from "../share"

it("renders a saved caption properly", () => {
  const share = renderer
    .create(<Share url="http://artsy.net/article/point-pencils" title="The Point of Pencils" />)
    .toJSON()
  expect(share).toMatchSnapshot()
})
