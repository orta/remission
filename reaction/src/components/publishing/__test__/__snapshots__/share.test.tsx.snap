// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a saved caption properly 1`] = `
.c3 {
  font-family: artsy-icons;
  color: white;
  font-size: 30px;
  margin: 0 5px;
  display: inline-block;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  max-width: 350px;
  margin-bottom: 40px;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  height: 40px;
  max-width: 113px;
  -webkit-text-decoration: none;
  text-decoration: none;
  background-color: black;
  color: white;
  border-radius: 2px;
  border: 1px solid black;
  margin-right: 10px;
}

.c1:hover {
  background-color: white;
}

.c1:hover .c2 {
  color: black;
}

<div
  className="c0"
>
  <a
    className="c1"
    href="https://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Fartsy.net%2Farticle%2Fpoint-pencils"
    onClick={[Function]}
    target="_blank"
  >
    <div
      className="c2 c3"
      style={undefined}
    >
      
    </div>
  </a>
  <a
    className="c1"
    href="https://twitter.com/intent/tweet?original_referer=http%3A%2F%2Fartsy.net%2Farticle%2Fpoint-pencils&text=The Point of Pencils&url=http%3A%2F%2Fartsy.net%2Farticle%2Fpoint-pencils&via=artsy"
    onClick={[Function]}
    target="_blank"
  >
    <div
      className="c2 c3"
      style={undefined}
    >
      
    </div>
  </a>
  <a
    className="c1"
    href="mailto:?subject=The Point of Pencils&body=Check out The Point of Pencils on Artsy: http://artsy.net/article/point-pencils"
    onClick={[Function]}
  >
    <div
      className="c2 c3"
      style={undefined}
    >
      
    </div>
  </a>
</div>
`;
